walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +121,68 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredGivenBusyOptional() {
```

Comment:
Could the test be named better?

Reply:
Sure. I will change it to "onlyMandatoryAttendeesAreConsidered". I think it is clearer for this test.